cmd/ipam-node/app/options/options.go

```diff
@@ -58,11 +58,12 @@ func New() *Options {
 // Options holds command line options for controller
 type Options struct {
 	cmdoptions.Options
-	MetricsAddr string
-	ProbeAddr   string
-	NodeName    string
-	BindAddress string
-	StoreFile   string
+	MetricsAddr    string
+	ProbeAddr      string
+	NodeName       string
+	PoolsNamespace string
```

Comment:
do we want to set default (kube-system) for PoolsNamespace to keep it in sync with controller?

Reply:
done